### PR TITLE
fix _OptionCheckBox reaction

### PR DESCRIPTION
**Pull request by 21pages**
#5217

### CODE REVIEW

File: flutter/lib/desktop/pages/desktop_setting_page.dart [hunk 1]
```diff
@@ -1671,13 +1671,11 @@ Widget _OptionCheckBox(BuildContext context, String label, String key,
   var ref = value.obs;
   onChanged(option) async {
     if (option != null) {
+      ref.value = option;
       if (reverse) option = !option;
```
Comment (rustdesk) [inline flutter/lib/desktop/pages/desktop_setting_page.dart:1675]: reverse ?

**Comment by rustdesk** [comment]
no need to waste time here.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5217

**Comment by SergeyMy** [comment]
@21pages 
why broke? doesn't work again!

https://github.com/rustdesk/rustdesk/blob/8d830382de5a80ef98b66aadabcc43b7aa88b94e/flutter/lib/desktop/pages/desktop_setting_page.dart#L1679

**Pull request merged**